test-cases/tests/multiAZ/p01-measure-downtime-during-down-az.md

```diff
@@ -45,71 +46,135 @@ services remain available.
 
    ```bash
    # e.g.
-   ./scripts/podsAz.sh redhat-rhmi-rhsso
-   Pods distribution for 'redhat-rhmi-rhsso'
+   ./scripts/podsAz.sh redhat-rhmi-user-sso
+   Pods distribution for 'redhat-rhmi-user-sso'
    | Pod name | Availability Zone |
    | -------- | ----------------- |
    | keycloak-0 | eu-west-1c |
    | keycloak-1 | eu-west-1b |
+
    ```
 
-6. Go to the OpenShift console, go through all the `redhat-rhmi-` prefixed namespaces and verify that all routes (Networking -> Routes) of RMOAN components are accessible
+5. Create customer-like application using customer-admin01 (or other user from dedicated-admin group)
+
+   ```bash
+   oc new-project httpbin
+   oc new-app jsmadis/httpbin
+   oc expose svc/httpbin
+   ```
 
-   > If some of the routes are not accessible, try again later. If they won't come up in the end, report the issue.
+6. Manage the customer-like app by 3scale and secure it by user SSO
 
-7. Clone [delorean](https://github.com/integr8ly/delorean) repo and run the following command to generate a downtime report using the delorean cli:
+   Probably the simplest way to do it is to run the performance test suite. The [branch](https://gitlab.cee.redhat.com/trepel/3scale-py-testsuite/-/tree/customer-like-app) has been created to simplify this as much as possible. If you ran the performance test suite before, you "just" have to clone the branch, update [settings.local.yaml](https://gitlab.cee.redhat.com/trepel/3scale-py-testsuite/-/blob/customer-like-app/config/settings.local.yaml) and execute the test suite
 
+   ```bash
+   pipenv run python -m pytest --performance_smoke testsuite/tests/performance/apicast/smoke/test_smoke_oidc.py
    ```
-   cd delorean
-   make build/cli
-   ./delorean pipeline query-report --config-file ./configurations/downtime-report-config-rhoam.yaml -o <output_dir>
+
+   If you haven't run the performance test suite before, see the comments in [MGDAPI-238](https://issues.redhat.com/browse/MGDAPI-238). And [Austin's doc](https://docs.google.com/document/d/1NJBUsieRkBLnN2PMAF5cpaH7uXq9mZCx1JQaT9Ruytk/edit?usp=sharing) in particular.
+
+   If you wish to do it manually, see the comments in [MGDAPI-198](https://issues.redhat.com/browse/MGDAPI-198).
+
+7. In terminal window #2, run the following [script](https://github.com/integr8ly/integreatly-operator/blob/master/scripts/alerts-during-perf-testing.sh)
+
+8. In terminal window #3, run a script to monitor the customer-like application
+
+   ```bash
+   #!/usr/bin/env bash
+
+   #run this like ./nameOfThisFile.sh 2>&1 | tee nameOfThisFileOutput.txt
+   while true; do
+     echo `date`
+     curl -isS -H 'Accept: application/json' -H "Authorization: Bearer <TOKEN>" https://<PRODUCTION-APICAST-URL> | grep HTTP | head -1
+     echo "----"
+     sleep 5;
+   done
    ```
```

Comment:
It is a little unclear where the `<TOKEN>` and the `<PRODUCTION-APICAST-URL>` are coming from. Can it be clarified

Reply:
never mind I read the next line

Reply:
Ok, probably not the best wording/structure ever but I leave it as it is for now.

Reply:
:+1: